4.analyze_data/notebooks/UMAP/scripts/0.UMAP.py

```diff
@@ -3,7 +3,7 @@
 
 # ## Extract UMAP embeddings for CFReT CP Features
 
-# In[1]:
+# In[5]:
```

Comment:
need to restart + run all

Reply:
Ah dang it great catch!

Reply:
I have done this and will rerun nbconvert.